README.md

```diff
@@ -96,34 +110,38 @@ cd SmartSim-Scaling
 pip install -r requirements.txt
 ```
 
+> Note that if you are using a Cray machine, you will need to run `CC=cc CXX=CC pip install -r requirements.txt`.
```

Comment:
Is this true?

Reply:
yes yes its bc Cray machines like to be different

Reply:
this is just so we can get mpi4py installed

Reply:
So why delete the installation instructions for mpi4py?

Reply:
also, why do you need `CC=cc CXX=CC` on pip install for Cray?